code/modules/admin/verbs/adminevents.dm

```diff
@@ -308,19 +308,75 @@
 		return
 
 	if(!holder.marked_datum || !istype(holder.marked_datum, /mob/living))
+		to_chat(usr, span_warning("Error: Please mark a mob to add actions to it."))
 		return
 
 	var/mob/living/marked_mob = holder.marked_datum
 
-	var/ability_type = input("Choose an ability", "Ability")  as null|anything in sort_list(subtypesof(/datum/action/cooldown/mob_cooldown), /proc/cmp_typepaths_asc)
+	var/list/all_mob_actions = sort_list(subtypesof(/datum/action/cooldown/mob_cooldown), /proc/cmp_typepaths_asc)
+
+	var/ability_type = tgui_input_list(usr, "Choose an ability", "Ability", all_mob_actions)
 
 	if(!ability_type)
 		return
 
-	var/datum/action/cooldown/mob_cooldown/add_ability = new ability_type()
+	var/datum/action/cooldown/mob_cooldown/add_ability
+
+	var/make_sequence = input(usr, "Would you like this to be in a sequence?", "Sequence Ability") as anything in list("Yes", "No")
```

Comment:
These `input`s don't have sanity checks for if the mob (or action, later on) is deleted between the input showing and entering - it may be wise to use some